src/heretic/r_things.c

```diff
@@ -520,7 +520,7 @@ void R_DrawVisSprite(vissprite_t * vis, int x1, int x2)
 
     colfunc = basecolfunc;
 #ifdef CRISPY_TRUECOLOR
-    blendfunc = I_BlendOverTinttab;
+    blendfunc = I_BlendWeakOverTinttab;
```

Comment:
Why not `I_BlendOverAltTinttab` here? I want to keep the strong/weak nomenclature out of the actual game code.